website/content/docs/security/acl/acl-tokens.mdx

```diff
@@ -145,7 +145,7 @@ Refer to the [API](/api-docs/acl/tokens) or [command line](/commands/acl/token)
 | `NodeIdentities`    | Specifies a list of node identities to apply to the token. See [Node Identities](/docs/security/acl/acl-roles#node-identities) in the "Roles" topic for additional information.                                                                            | Array     | none           |
 | `Legacy`            | Indicates if the token was created using the the legacy ACL system.                                                                                                                                                                                        | Boolean   | `false`        |
 | `Policies`          | List of policies linked to the token, including the policy ID and name.                                                                                                                                                                                    | String    | none           |
-
+| `Roles`             | List of roles linked to the token, including the role ID and name.                                                                                                                                                                                      | String    | none           |
```

Comment:
```suggestion
| `Roles`             | List of roles linked to the token, including the role ID and name.                                                                                                                                                                                      | String    | none           |

```

Reply:
I'm failing at "spot the difference" here

Reply:
Oh I added a newline 😆

Reply:
lol, :doh: